torch/distributed/rpc/api.py

```diff
@@ -201,6 +202,8 @@ def shutdown(graceful=True):
     ``graceful=False``, this is a local shutdown, and it does not wait for other
     RPC processes to reach this method.
 
+    Warning, future.wait() should not be called after shutdown().
```

Comment:
Shall we detect this in the unwrap function?

Reply:
See https://github.com/pytorch/pytorch/pull/35154#discussion_r419858448.